src/keri/app/clienting.ts

```diff
@@ -232,19 +233,23 @@ export class SignifyClient {
     /**
      * Fetch a resource from from an external URL with headers signed by an AID
      * @async
+     * @param {string} aidName Name or alias of the AID to be used for signing
      * @param {string} url URL of the resource
      * @param {string} path Path to the resource
-     * @param {string} method HTTP method
-     * @param {any} data Data to be sent in the body of the resource
-     * @param {string} aidName Name or alias of the AID to be used for signing
+     * @param {RequestInit} req Request options should include:
+     *     - method: HTTP method
+     *     - data Data to be sent in the body of the resource.
+     *              If the data is a CESR JSON string then you should also set contentType to 'application/json+cesr'
+     *              If the data is a FormData object then you should not set the contentType and the browser will set it to 'multipart/form-data'
+     *              If the data is an object then you should use JSON.stringify to convert it to a string and set the contentType to 'application/json'
+     *     - contentType Content type of the request.
      * @returns {Promise<Response>} A promise to the result of the fetch
      */
     async signedFetch(
+        aidName: string,
         url: string,
         path: string,
```

Comment:
```suggestion
        url: string,
```

I personally don't see a value in separating these args. In fetch which is a familiar API you do not separate the host and the path. They are both part of the URL.

Reply:
To get the for the signature you can simply do `const pathname = new URL(url).pathname`. This also strips the query params so the `path.slit["?"]` is not needed on L272

Reply:
I think whoever defined the method signature was trying to explicitly align with https://datatracker.ietf.org/doc/html/rfc9421#name-path but i'm okay with aligning with fetch and having the method handle the url parts as you specified. Let me know if the RFC alignment alters your thoughts.

Also, browser extensions that use signify-ts should sign headers the same way as signify-ts, but they don't do the fetch. So I'll refactor this method to take a url but not do the fetch. Let me know if you have thoughts on that.

Reply:
Yeah, good idea. I think that makes sense. So you basically return a "signed" RequestInit? Imo, that makes it much more useful.

If you make sure the `createSignedRequest` returns a `Request` (available as a type in the global namespace in both node.js and browsers), the following usage would be supported and be very  straightforward for anyone to use.


```typescript
   // SignifyClient
   async createSignedRequest(aidName: string, url: string, requestInit: RequestInit): Promise<Request> {
   }
  

const request = await client.createSignedRequest("aidname", url, initialRequestInit);
await fetch(request)
```

Reply:
> Yeah, good idea. I think that makes sense. So you basically return a "signed" RequestInit? Imo, that makes it much more useful.
> 
> If you make sure the `createSignedRequest` returns a `Request` (available as a type in the global namespace in both node.js and browsers), the following usage would be supported and be very straightforward for anyone to use.
> 
> ```ts
>    // SignifyClient
>    async createSignedRequest(aidName: string, url: string, requestInit: RequestInit): Promise<Request> {
>    }
>   
> 
> const request = await client.createSignedRequest("aidname", url, initialRequestInit);
> await fetch(request)
> ```

great! working on  that update now